src/lib/core/components/computations/PassThroughComputation.tsx

```diff
@@ -32,7 +33,7 @@ const visualizationTypes: Record<string, VisualizationType> = {
   twobytwo: twoByTwoVisualization,
   conttable: contTableVisualization,
   scatterplot: scatterplotVisualization,
-  // lineplot: scatterplotVisualization,
+  timeseries: lineplotVisualization,
```

Comment:
```suggestion
  lineplot: lineplotVisualization, // Time series
```

Reply:
thanks @bobular . that should resolve w the new backend changes. also explains why you were seeing something different to me. im running the data service branch locally as well.

Reply:
i suppose now ive tested it a bit i could merge the data service work.. but was kind of hoping to merge these together given the api changes. if what you suggest above is enough to get things testable for you, maybe thats fine for now?

Reply:
Actually I didn't manage to get any plots working with my minor change - but I haven't looked into why - as I wasn't sure it was ready for that kind of testing (I realise now it is) - I'll have another look tomorrow.